### ISSUE TITLE
App crashes on iOS when using BottomSheetModal on Production

### ISSUE THREAD

**Issue by Bossmoss16** [gorhom/react-native-scroll-bottom-sheet#1275]
# Bug

hello,

I use Expo and I want to Test my App in Production so I execute This command

`Npx expo Start —no-dev —minify` 

When I Open The app its crashed without errors. So I debug it and I noticed when I remove bottom sheet it works And my app not crashed. So on android it works but on iOS its crash when I use. Anyone else faced this issue ?

## Environment info

| Library                         | Version |
| ------------------------------- | ------- |
| @gorhom/bottom-sheet            | 4.4.0  |
| react-native                    |  0.70.5  |
| expo                                 | 47 |
| react-native-reanimated         | 2.9.1 |
| react-native-gesture-handler    | 2.9.0 |

## Steps To Reproduce

1. run expo production command
2. opened via real device iphone

Describe what you expected to happen:

1. crashed without error

## Reproducible sample code
```

import React, { useCallback, useMemo, useRef } from 'react';
import { View, Text, StyleSheet, Button } from 'react-native';
import {
  BottomSheetModal,
  BottomSheetModalProvider,
} from '@gorhom/bottom-sheet';

const App = () => {
  // ref
  const bottomSheetModalRef = useRef<BottomSheetModal>(null);

  // variables
  const snapPoints = useMemo(() => ['25%', '50%'], []);

  // callbacks
  const handlePresentModalPress = useCallback(() => {
    bottomSheetModalRef.current?.present();
  }, []);
  const handleSheetChanges = useCallback((index: number) => {
    console.log('handleSheetChanges', index);
  }, []);

  // renders
  return (
    <BottomSheetModalProvider>
      <View style={styles.container}>
        <Button
          onPress={handlePresentModalPress}
          title="Present Modal"
          color="black"
        />
        <BottomSheetModal
          ref={bottomSheetModalRef}
          index={1}
          snapPoints={snapPoints}
          onChange={handleSheetChanges}
        >
          <View style={styles.contentContainer}>
            <Text>Awesome 🎉</Text>
          </View>
        </BottomSheetModal>
      </View>
    </BottomSheetModalProvider>
  );
};

const styles = StyleSheet.create({
  container: {
    flex: 1,
    padding: 24,
    justifyContent: 'center',
    backgroundColor: 'grey',
  },
  contentContainer: {
    flex: 1,
    alignItems: 'center',
  },
});

export default App
```;

**Comment by Yassdrk** [comment]
me too in development !

**Comment by shop-fluencer** [comment]
still exists

**Comment by jaronfort** [comment]
I see that this issue is closed but I am having the same issue in Expo Go. Everything works perfectly in development. However, in production after installing via TestFlight, the app crashes when navigating to the screen that contains the bottom sheet. I removed the code for the bottom sheet and crashes halted. I added the bare minimum (i.e., a bottom sheet and a ref with no state or functionality). This caused the crashes to continue.
